utils/tlsconfig/tlsconfig.go

```diff
@@ -13,7 +13,7 @@ func Get() *tls.Config {
 			tls.TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
 			tls.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
 			tls.TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
-			tls.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256,
+			tls.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256, //nolint:gosec
```

Comment:
Minor suggestion: Maybe as a separate task, we can introduce https://github.com/golangci/golangci-lint/tree/master/pkg/golinters/nolintlint

It helps to describe the reason for `//nolint` rules to provide some context (e.g why only this specific config needs a nolint directive).